Unity Project- Kaiju/Assets/Scripts/Gameplay/Health/EnemyHealth.cs

```diff
@@ -0,0 +1,34 @@
+using UnityEngine;
```

Comment:
Same class as PlayerHealth. Could be an inherited class, can even be an interface.